core/tests/test_utils.py

```diff
@@ -165,6 +196,9 @@ def log_line(self, line):
         print '%s%s' % (LOG_LINE_PREFIX, line)
 
     def _delete_all_models(self):
+        """To be overriden the inherited Test class. Shall be used to delete
```

Comment:
Deletes all keys from the NDB datastore. Subclasses of TestBase should override this method.